spec/index.bs

```diff
@@ -1375,6 +1375,7 @@ implicitly [$valid to use with|unusable$].
 
     1. Let |features| be the [=set=] of values in
         |descriptor|.{{GPUDeviceDescriptor/requiredFeatures}}.
+    1. [=set/Prepend=] {{GPUFeatureName/"core-features-and-limits"}} to |features|.
```

Comment:
Given @Kangz feedback in #5148 (which I agree with) I think this is better off being simply appended rather than prepended? I'd also like to see the list sorted so that the exact position doesn't matter, but that can be a follow up PR after group discussion.

Reply:
Sure, the spec'd order is kinda bogus anyway and I'm pretty sure at least chrome is not implementing it, so it's an arbitrary choice until we fix it.